echopype/tests/convert/test_parsed_to_zarr.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from echopype import open_raw
+
+
+@pytest.fixture
+def ek60_path(test_path):
+    return test_path['EK60']
+
+
+@pytest.fixture
+def ek80_path(test_path):
+    return test_path['EK80']
+
+
+def compare_zarr_vars(ed_zarr, ed_no_zarr, var_to_comp, ed_path):
+    # TODO: add docstring and comments
+
+    for var in var_to_comp:
+
+        for chan in ed_zarr[ed_path][var].channel:
+
+            # here we compute to make sure values are being compared, rather than just shapes
+            var_zarr = ed_zarr[ed_path][var].sel(channel=chan).compute()
+            var_no_zarr = ed_no_zarr[ed_path][var].sel(channel=chan)
+
+            assert var_zarr.identical(var_no_zarr)
+
+    ed_zarr[ed_path] = ed_zarr[ed_path].drop(var_to_comp)
+    ed_no_zarr[ed_path] = ed_no_zarr[ed_path].drop(var_to_comp)
+    return ed_zarr, ed_no_zarr
+
+
```

Comment:
Instead of xfail, let's do skip
```suggestion
@pytest.mark.skip(reason="Full testing of writing variables directly to a zarr store has not been implemented yet.")
```